source/wp-content/themes/wporg-news-2021/sass/post/_header.scss

```diff
@@ -6,12 +6,14 @@
 	.wp-block-post-author,
 	.wp-block-post-terms {
 		display: inline-block;
+		margin-top: 0 !important; // TODO this might be fixable via the blockGap prop in theme.json.
```

Comment:
not sure if it'd work here, but the header didn't need the `!important`, just:

```css
.wp-block-group.site-header {
	margin-top: 0;
}
```

Reply:
i think this might work universally

```css
[class*="wp-container-"] .site-footer__logos-container {
    margin-top: 0;
}
```

xref #25

Reply:
What do you mean universally?

Reply:
sorry, i meant we might always be able to do 

```css
[class*="wp-container-"] .{ whatever element you want to target } {
    margin-top: 0;
}
```

so instead of the `!important` rules we'd have

```css
[class*="wp-container-"] .wp-block-post-author,
[class*="wp-container-"] .wp-block-post-terms,
[class*="wp-container-"] .wp-block-post-title {
    margin-top: 0;
}
```

Reply:
in some cases it might be better to override the gap for specific blocks in `theme.json`, though?

xref https://github.com/WordPress/gutenberg/pull/33991

Reply:
Ah, I see. This definitely sounds better than using `!important`.

> in some cases it might be better to override the gap for specific blocks in theme.json, though?

I would expect it would only make sense to use theme.json if you _always_ want the gap to be different, regardless of what context the block is in.

Reply:
Removed the `!important` here in 546e280